### DISCUSSION TITLE
Supporting new GitHub `note` syntax

### DISCUSSION THREAD

**Discussion by afbora** [thephpleague/commonmark#1019] [Q&A] [answered]
I've tested new GitHub `[!NOTE]` syntax with `GithubFlavoredMarkdownExtension` but I couldn't achieve it. Is this library support the new GitHub syntax?

````md
> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

> [!TIP]
> Optional information to help a user be more successful.

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

> [!CAUTION]
> Negative potential consequences of an action.
````

https://github.com/orgs/community/discussions/16925

**Comment by colinodell** [comment, marked as answer]
No, we don't support that today.  Please see https://github.com/thephpleague/commonmark/issues/1012 for more information.  TL;DR: It's not technically part of GFM.
